demo/commerce/play-shop-discover-to-ordercloud-feed-importer/.eslintrc.js

```diff
@@ -0,0 +1,35 @@
+module.exports = {
```

Comment:
Could you please add the `.vscode` folder to this project with similar settings as the other projects?